scripts/globals/status.lua

```diff
@@ -797,45 +802,48 @@ dsp.effect =
 };
 
 ----------------------------------
--- SC masks
+-- SC masks (these are not currently used anywhere in code base. do we need them?)
```

Comment:
They *used to be* for a thing removed ages ago, IIRC. I dunno if we have any future thing we might use them for but I spoze we could always re-add them then. Also doesn't hurt to just leave them commented out for now.